packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/ChatbotMessage/MessageWithQuickResponses.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Message from '@patternfly/virtual-assistant/dist/dynamic/Message';
+import patternflyAvatar from './patternfly_avatar.jpg';
+
+export const MessageWithQuickResponsesExample: React.FunctionComponent = () => (
+  <>
+    <Message
+      name="Bot"
+      role="bot"
+      avatar={patternflyAvatar}
+      content="Did you clear your cache"
+      quickResponses={[
+        { id: '1', content: 'Yes', onClick: () => alert('Clicked yes') },
+        { id: '2', content: 'No', onClick: () => alert('Clicked no') }
+      ]}
+    />
+    <Message
+      name="Bot"
+      role="bot"
+      avatar={patternflyAvatar}
+      content="What browser are you noticing the issue in?"
+      quickResponses={[
+        { id: '1', content: 'Microsoft Edge', onClick: () => alert('Clicked Edge') },
+        { id: '2', content: 'Google Chrome', onClick: () => alert('Clicked Chrome') },
+        { id: '3', content: 'Mozilla Firefox', onClick: () => alert('Clicked Firefox') },
+        { id: '4', content: 'Apple Safari', onClick: () => alert('Clicked Safari') },
+        { id: '5', content: 'Internet Explorer', onClick: () => alert('Clicked Internet Explorer') }
+      ]}
+    />
+    <Message
+      name="Bot"
+      role="bot"
+      avatar={patternflyAvatar}
+      content="Welcome back! Ready to continue?"
```

Comment:
```suggestion
      content="Welcome back! How can I help?"
```
This also may be outside of the scope of this pr, just wondering if this is a better question to ask to create more of a "call and response" relationship with the phrasing in the labels

Or something like "What can I help you with today?"

Reply:
Just updated!